globalnoc-tsds-datasource/src/partials/query.editor.html

```diff
@@ -32,9 +32,12 @@
               <option "style = background: #292929" value="max">max</option>
               <option "style = background: #292929" value="min">min</option>
               <option "style = background: #292929" value="percentile">percentile</option>
+	      <option "style = background: #292929" value="template">template</option>
             </select>
           </span>
 
+	  <input type="text" ng-if="ctrl.target.aggregator[$index]=='template'" class="gf-form-input width-21" placeholder="$Variable" ng-model="ctrl.target.templateVariableValue[$index]"/>
```

Comment:
Why is placeholder="$Variable" here? Unless I am misreading it, I don't think that actually does anything and is a bit misleading. I'd change this to "template variable name" or something that similar that fits.